config/networks.json

```diff
@@ -526,6 +526,23 @@
     "safeAddress": "0xa89a87986e8ee1Ac8fDaCc5Ac91627010Ec9f772",
     "gasZipChainId": 249
   },
+  "worldchain": {
+    "name": "World Chain",
+    "chainId": 480,
+    "nativeAddress": "0x0000000000000000000000000000000000000000",
+    "nativeCurrency": "ETH",
+    "wrappedNativeAddress": "0x4200000000000000000000000000000000000006",
+    "status": "inactive",
+    "type": "mainnet",
+    "rpcUrl": "https://worldchain-mainnet.g.alchemy.com/public",
+    "verificationType": "etherscan",
+    "explorerUrl": "https://worldscan.org",
+    "explorerApiUrl": "https://api.worldscan.org/api",
+    "multicallAddress": "0xcA11bde05977b3631167028862bE2a173976CA11",
+    "safeApiUrl": "https://safe-transaction-worldchain.safe.global/api",
+    "safeAddress": "0xa6E64bC059ddAa72ec374C50B1A5c6b3a27d64Ce",
```

Comment:
the newly added property `safeWebUrl` is missing, here an example from Arbitrum:

"safeWebUrl": "https://app.safe.global/home?chain=oeth&safe=arb1:0x9e606d0d2BbA344b911e2F4Eab95d9235A83fe15",

Reply:
updated